src/AlignLoads.h

```diff
@@ -13,7 +13,13 @@ namespace Internal {
 /** Attempt to rewrite unaligned loads from buffers which are known to
  * be aligned to instead load aligned vectors that cover the original
  * load, and then slice the original load out of the aligned
- * vectors. */
+ * vectors.
+ *
+ * Types that are less than min_bytes_to_align in size do not have
```

Comment:
Nit: I would change "do not have alignment applied" to "are not rewritten", to reference the wording in the paragraph above.